runtimes/web/src/index.js

```diff
@@ -278,7 +334,6 @@ async function loadCartWasm () {
             const DPAD_MAX_DISTANCE = 100;
             const DPAD_DEAD_ZONE = 10;
             const BUTTON_MAX_DISTANCE = 50;
-            const DPAD_ACTIVE_ZONE = 3 / 5; // cos of active angle, greater that cos 60 (1/2)
```

Comment:
Did you mean to revert these virtual gamepad changes?

Reply:
ahhh. I synced wrong version (i had a dPAD in isolated branch, but commit onto main), wait.